draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -157,7 +157,7 @@ All Domain Control Validation mechanisms are implemented by a DNS resource recor
 
 The RECOMMENDED method of doing DNS-based domain control validation is to use DNS TXT records as the Validation Record. The name is constructed as described in {{name}}, and RDATA MUST contain at least a Random Token (constructed as in {{random-token}}). If there are multiple RDATA strings for a record, the Application Service Provider MUST treat them as a concatenated string. If metadata (see {{metadata}}) is not used, then the unique token generated as-above can be placed as the only contents of the RDATA. For example:
 
-    _service-challenge.example.com.  IN   TXT  "3419...3d206c4"
+    _examplename-challenge.example.com.  IN   TXT  "3419...3d206c4"
```

Comment:
maybe _example_service would be clearer ?

Reply:
Yes, that's better. I don't have edit privs here, so please make that change throughout.